src/scene/camera.js

```diff
@@ -203,21 +204,16 @@ Object.assign(pc, function () {
             return this._clearOptions;
         },
 
-        /**
-         * @private
-         * @function
-         * @name pc.Camera#getProjectionMatrix
-         * @description Retrieves the projection matrix for the specified camera.
-         * @returns {pc.Mat4} The camera's projection matrix.
-         */
-        getProjectionMatrix: function () {
+        _evaluateProjectionMatrix: function () {
             if (this._projMatDirty) {
                 if (this._projection === pc.PROJECTION_PERSPECTIVE) {
                     this._projMat.setPerspective(this._fov, this._aspect, this._nearClip, this._farClip, this._horizontalFov);
+                    this._projMatSkybox.copy(this._projMat);
                 } else {
                     var y = this._orthoHeight;
                     var x = y * this._aspect;
                     this._projMat.setOrtho(-x, x, -y, y, this._nearClip, this._farClip);
+                    this._projMatSkybox.setPerspective(90, this._aspect, this._nearClip, this._farClip);

```

Comment:
Is fixing this to 90 degrees limiting at all? Would a developer expect to be able to tweak this? Maybe using `this._fov` instead?

Reply:
I considered this as well, but didn't like to expose control without Editor support. I added it now .. you can change to Perspective, move slider, and back to Ortho and skybox is unchanged, so that's great.
Perhaps we should update editor to leave FOV displayed? Perhaps rename it to "Skybox Field of View" if Ortho mode is selected.